pkg/sql/compile/compile.go

```diff
@@ -2963,3 +2969,34 @@ func (c *Compile) newInsertMergeScope(arg *insert.Argument, ss []*Scope) *Scope
 	}
 	return c.newMergeScope(ss2)
 }
+
```

Comment:
Either a session variable, so that you can set using sql

set @check_rc_invalid_error = 1;

Or, just use one of our config.    Because you pick it up only on startup therefore os env does not really offer anything better than our own config.

Reply:
> Either a session variable, so that you can set using sql
> 
> set @check_rc_invalid_error = 1;
> 
> Or, just use one of our config. Because you pick it up only on startup therefore os env does not really offer anything better than our own config.

Actually I think OS env is a VERY bad idea.    Too hard to manage, repro nightmare.

Reply:
sessino variable can not used in txn client, If we need to use it, the changes are a bit much. So add environment variables. This one is also a temporary one and will remove these checks later on

Reply:
the problem is that when you ask others to repro, they need to set env variable.    depending on how they login ssh etc, it is really hard to track if the variable is set or not, really confusing.

if it is a config in our cn.toml file, at least I can take a look at the config file and know if I have set it or not.